lib/dice.js

```diff
@@ -206,6 +206,18 @@ class DiceObject {
 
             geometry.faces[i].materialIndex = materialIndex + 1;
         }
+        if (this.values == 4 && toValue != fromValue) {
```

Comment:
@liffiton, I see this working in most cases, but it fails where the ` && toValue != fromValue` condition is met. Removing that restriction from the `if` seems to work in all cases that I have observed.
```suggestion
        if (this.values == 4) {
```